pvlib/atmosphere.py

```diff
@@ -533,3 +534,165 @@ def angstrom_alpha(aod1, lambda1, aod2, lambda2):
+
+
+# Values of the Hellmann exponent
+HELLMANN_SURFACE_EXPONENTS = {
+    'unstable_air_above_open_water_surface': 0.06,
+    'neutral_air_above_open_water_surface': 0.10,
+    'stable_air_above_open_water_surface': 0.27,
+    'unstable_air_above_flat_open_coast': 0.11,
+    'neutral_air_above_flat_open_coast': 0.16,
+    'stable_air_above_flat_open_coast': 0.40,
+    'unstable_air_above_human_inhabited_areas': 0.27,
+    'neutral_air_above_human_inhabited_areas': 0.34,
+    'stable_air_above_human_inhabited_areas': 0.60,
+}
+
+
+def windspeed_hellmann(wind_speed_reference, height_reference,
+                       height_desired, exponent=None,
+                       surface_type=None):
+    r"""
+    Estimate wind speed for different heights.
+
+    The model is based on the power law equation by Hellmann [1]_ [2]_.
+
+    Parameters
+    ----------
+    wind_speed_reference : numeric
+        Measured wind speed. [m/s]
+
+    height_reference : float
+        The height at which the wind speed is measured. [m]
+
+    height_desired : float
+        The height at which the wind speed will be estimated. [m]
+
+    exponent : float, optional
+        Exponent based on the surface type. [-]
+
+    surface_type : string, optional
+        If supplied, overrides ``exponent``. Can be one of the following
+        (see [1]_):
+
+        * ``'unstable_air_above_open_water_surface'``
+        * ``'neutral_air_above_open_water_surface'``
+        * ``'stable_air_above_open_water_surface'``
+        * ``'unstable_air_above_flat_open_coast'``
+        * ``'neutral_air_above_flat_open_coast'``
+        * ``'stable_air_above_flat_open_coast'``
+        * ``'unstable_air_above_human_inhabited_areas'``
+        * ``'neutral_air_above_human_inhabited_areas'``
+        * ``'stable_air_above_human_inhabited_areas'``
+
+    Returns
+    -------
+    wind_speed : numeric
+        Adjusted wind speed for the desired height. [m/s]
+
+    Raises
+    ------
+    ValueError
+        If neither of ``exponent`` nor a ``surface_type`` is given.
+        If both ``exponent`` and a ``surface_type`` is given. These parameters
+        are mutually exclusive.
+
+    KeyError
+        If the specified ``surface_type`` is invalid.
+
+    Notes
+    -----
+    The equation for calculating the wind speed at a height of :math:`h` is
+    given by the following power law equation [1]_ [2]_:
+
+    .. math::
+       :label: wind speed
+
+        U_{w,h} = U_{w,ref} \cdot \left( \frac{h}{h_{ref}} \right)^a
+
+    where :math:`h` [m] is the height at which we would like to calculate the
+    wind speed, :math:`h_{ref}` [m] is the reference height at which the wind
+    speed is known, and :math:`U_{w,h}` [m/s] and :math:`U_{w,ref}`
+    [m/s] are the corresponding wind speeds at these heights. :math:`a` is a
+    value that depends on the surface type. Some values found in the literature
+    [1]_ for :math:`a` are the following:
```

Comment:
```suggestion
        WS_{h} = WS_{ref} \cdot \left( \frac{h}{h_{ref}} \right)^a

    where :math:`h` [m] is the height at which we would like to calculate the
    wind speed, :math:`h_{ref}` [m] is the reference height at which the wind
    speed is known, and :math:`WS_{h}` [m/s] and :math:`WS_{ref}`
    [m/s] are the corresponding wind speeds at these heights. The exponent
    :math:`a` [unitless] depends on the surface type. Some values found in the
    literature [1]_ for :math:`a` are:
```

Reply:
OK to not specify units of m/s for `wind_speed_reference`, but a comment that `wind_speed` has the same units as `wind_speed_reference` would be useful.